Makefile

```diff
@@ -1,40 +1,17 @@
 # Documentation
-docs/tuist/preview:
-	./make/tasks/docs/tuist/preview.sh
-docs/tuist/build:
-	./make/tasks/docs/tuist/build.sh
+docs/preview:
+	./make/tasks/docs/preview.sh
+docs/build:
+	./make/tasks/docs/build.sh
 
 # Tuist
 tuist/edit:
 	./make/tasks/tuist/edit.sh
-tuist/fetch:
-	./make/tasks/tuist/fetch.sh $(ARGS)
 tuist/generate:
 	./make/tasks/tuist/generate.sh $(ARGS)
-tuist/cache-warm:
-	./make/tasks/tuist/cache-warm.sh $(ARGS)
-tuist/run:
```

Comment:
What is this needed for? We should rather expose some makefile rule for the individual runs, removing it for now